Neos.ContentRepository.Core/Classes/Feature/WorkspaceRebase/CommandThatFailedDuringRebase.php

```diff
@@ -15,19 +15,20 @@
 namespace Neos\ContentRepository\Core\Feature\WorkspaceRebase;
 
 use Neos\ContentRepository\Core\CommandHandler\CommandInterface;
+use Neos\EventStore\Model\Event\SequenceNumber;
 
 /**
- * @internal implementation detail of WorkspaceCommandHandler
+ * @api part of the exception exposed when rebasing failed
  */
 final readonly class CommandThatFailedDuringRebase
 {
     /**
-     * @param int $sequenceNumber the event store sequence number of the event containing the command to be rebased
+     * @param SequenceNumber $sequenceNumber the event store sequence number of the event containing the command to be rebased
      * @param CommandInterface $command the command that failed
      * @param \Throwable $exception how the command failed
      */
     public function __construct(
-        public int $sequenceNumber,
+        public SequenceNumber $sequenceNumber,
```

Comment:
todo this is api and will be interpreted in the neos ui, to we want to pass the `Neos\EventStore\Model\Event\SequenceNumber` to the outside?

see use: https://github.com/neos/neos-ui/blob/7468ffb851ac16713f8afda9277204a396f09a1c/Classes/Infrastructure/ContentRepository/ConflictsFactory.php#L118

Reply:
The CR is already "only" a user of the SequenceNumber, I find it a bit smelly to hand this around, but then it's not exactly internal to the CR either.

Reply:
made it `@internal` and removed use in Neos ui ;) https://github.com/neos/neos-ui/pull/3769/commits/15c13ab601a44b54f178304558e48a58cb7ef1b8